packages/lesswrong/components/common/Logo.jsx

```diff
@@ -8,7 +8,7 @@ const styles = theme => ({
   }
 })
 
-const Logo = ({classes}) => <img
+const Logo = ({classes}) => getSetting('logoUrl', null) && <img
```

Comment:
I like this pattern better tbh, but then you should remove the getSetting && logo in the Header.